CMakeLists.txt

```diff
@@ -71,8 +71,6 @@ cmake_dependent_option(CHAI_ENABLE_DOCS "Build CHAI docs" On
                        "ENABLE_DOCS" Off)
 cmake_dependent_option( CHAI_ENABLE_GMOCK "Build CHAI with gmock" On
                         "ENABLE_GMOCK" Off )
-cmake_dependent_option(CHAI_ENABLE_REPRODUCERS "Build CHAI reproducers" On
```

Comment:
ENABLE_REPRODUCERS is not a variable defined anywhere (unlike ENABLE_TESTS for example, which is a BLT variable). Even if it was, I don't think it makes sense for it to be a dependent variable. It is already an option in cmake/SetupChaiOptions.cmake.